devicon.json

```diff
@@ -9111,5 +9111,35 @@
         },
         "color": "#2684FF",
         "aliases": []
+    },
+    {
+        "name": "pfsense",
+        "altnames": [
+            "pfSense"
+        ],
+        "tags": [
+           "network"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "original",
+                "original-wordmark"
+            ]
+        },
+        "color": "#000000",
```

Comment:
Couldn't find an official black icon, but the one on wikimedia is `#1a1a1a` instead of `#000000`
```suggestion
        "color": "#1a1a1a",
```

Reply:
Their SVG icon doesn't have any fill attribute, resulting in `#000000` color icon.
That's why I chose this color instead of Wikimedia's. 🤔

Reply:
Aha, I see!
Didn't see that they had an icon, mb!

I looked through their repo now though, and found that they do in fact have a `original` version here: https://github.com/pfsense/pfsense/blob/a7086b04cae21ca742fdeefd1019ee1401b6dded/src/usr/local/www/safari-pinned-tab.svg
![](https://raw.githubusercontent.com/pfsense/pfsense/a7086b04cae21ca742fdeefd1019ee1401b6dded/src/usr/local/www/safari-pinned-tab.svg)

So we should probably use that

Reply:
I didn't open that one! 🥲
Better late than never! 😅